packages/core/EventManager.ts

```diff
@@ -329,11 +329,10 @@ export default class EventManager {
        *  Not ideal but, if we don't find a destination calendar,
        * fallback to the first connected calendar
        */
-      const [credential] = this.calendarCredentials;
-      if (!credential) {
-        return [];
+      const [credential] = this.calendarCredentials.filter((cred) => cred.type === "calendar");
```

Comment:
Hmm, actually it feels weird that the array is called `CALENDARcredentials`, but still potentially could include a credential that is not of type `calendar`. Should we explain this somewhere in a comment or rename the property?

Reply:
well, they are all calendar credentials, for apps that manage events, the thing is that we actually treat calendar apps per se differently from apps that are not calendars but manage events either way, like CRMs

I guess we can explain it better in the code, yeah